src/App.js

```diff
@@ -0,0 +1,74 @@
+import logo from "./logo.svg";
+import "./App.css";
+import "./index.css";
+import ironhacklogo from "./images/ironhack-logo-xs.png";
+import hamburguer from "./images/menu-top-xs.png";
+import item1 from "./images/icon1.png";
+import item2 from "./images/icon2.png";
+import item3 from "./images/icon3.png";
+import item4 from "./images/icon4.png";
+
+function App() {
+  return (
+    <div className="padding">
+      <div className="navbar">
+        <img
+          src={ironhacklogo}
+          alt="example"
```

Comment:
Here i would put something more meanfull for example
```suggestion
          alt="ironhack-logo"
```